apps/src/applab/designElements/button.jsx

```diff
@@ -160,15 +161,7 @@ class ButtonEvents extends React.Component {
 
   getClickEventCode() {
     const id = elementUtils.getId(this.props.element);
-    const code =
-      'onEvent("' +
-      id +
-      '", "click", function(event) {\n' +
-      '  console.log("' +
```

Comment:
Is it okay that the `generateEventHandler` function doesn't generate these console logs? (here and elsewhere)

Reply:
yep, i've removed the console.logs and the default event param to make the design elements match the `onEvent` block

Reply:
Sorry maybe I'm missing something obvious, but I thought the goal was just to remove the event param? Why do we also want to remove the function body?

Reply:
re-reading hannah's slack and the forum post -- you might be right, they only mention the event param. let me ask hannah which is preferred

Reply:
hannah confirmed that just removing the event parameter is good. thank you for triple-checking me lol